src/csrs/service/csrsService.ts

```diff
@@ -18,17 +18,117 @@ export class CsrsService {
 		this.cacheService = cacheService
 	}
 
+	async editDescription(data: any, user: any) {
+		return await this.restService.postWithoutFollowingWithConfig('api/quiz/update',
+			data,
+			{
+				headers: {
+					Authorization: `Bearer ${user.accessToken}`,
```

Comment:
Best to have a helper method like getHeader() that returns the header object

Reply:
Yes. Have a utility method

Reply:
Will add it 👍